charactersheet/charactersheet/viewmodels/common/wizard_steps/wizard_profile_step.js

```diff
@@ -65,6 +71,29 @@ function WizardProfileStepViewModel() {
         return traits;
     };
 
+    self.populateBackpackItems = function() {
+        var items = [];
+        var populatedItems = [];
+        var backpack = self.backpack();
+        Object.keys(DataRepository.backpacks).forEach(function(key) {
+            //Obtain items array for matching backpack
+            if (key == backpack) {
+                items = DataRepository.backpacks[key];
+            }
+        });
+
+        if (items) {
+            items.forEach(function(element, idx, _) {
+                var item = DataRepository.items[element.name];
+                //Add item quantity from backpacks object
+                item.itemQty = element.count;
```

Comment:
Yes, the backpacks json doc contains the count of how many inventory items need to be added. So here we are overriding the default count with the actual count of items required per the backpack json.